lib/openhab/core/dsl/monkey_patch/ruby/number.rb

```diff
@@ -26,6 +26,15 @@ module IntegerExtensions
             alias ms millis
             alias minute minutes
             alias hour hours
+
+            #
+            # Create an OpenHAB PercentType
+            #
+            # @return [Java::OrgOpenhabCoreLibraryTypes::PercentType] PercentType corresponding to the value
+            #
+            def percent
```

Comment:
Do we want to define % as well? 

Not sure we do, but what do you think?

Reply:
Wouldn't that mess with the modulus operator?

Reply:
Not sure I do either...  Something about 'percent' however feels off from the rest of the DSL. 
Should we think of this like a quantity type? 

50 | '%' 

?

Reply:
Not sure, we would have to override the command method to translate the quantitytype to a percenttype before passing it to the eventbus, otherwise the command won't be accepted, even when converted to string.

Reply:
Another solution would be to skip this entirely, since sending regular Integers as command works as well.

Reply:
Let's remove percent for now?  Since we aren't sure and I would rahter not remove it as a breaking change.